File: server/icecreamapp/src/main/java/com/ssafy/icecreamapp/model/dto/Icecream.java
package com.ssafy.icecreamapp.model.dto;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

@Getter
@Setter
@ToString
@NoArgsConstructor
public class Icecream {
    int id;
    String name;
    int price;
    int isEvent;
    String type;
    int kcal;
    String img;
    String content;
    int count;
}

File: android/app/src/main/java/com/ssafy/icepop/data/model/dto/IceCream.kt
package com.ssafy.icepop.data.model.dto

data class IceCream(
    val id: Int,
    val name: String,
    val content: String,

    val price: Int,
    val count: Int,
    val isEvent: Int,
    val kcal: Int,

    val type: String,
    val img: String
)